.github/workflows/InitDev,Unit tests,Integration tests,Package.yml

```diff
@@ -135,4 +135,4 @@ jobs:
         uses: actions/upload-artifact@26f96dfa697d77e81fd5907df203aa23a56210a8 # v4.3.0
         with:
           name: NugetPackages.Artifacts
-          path: ${{ github.workspace }}/Ed-Fi-ODS-Implementation/NugetPackages/*.nupkg
+          path: ${{ github.workspace }}/Ed-Fi-ODS-Implementation/**/*.nupkg
```

Comment:
This change can be reverted as per this line https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/pull/1026/files#diff-935c1249167bc4db15f47cd61ce30d7460a393c53a1a6eed4a4318dd95b9b993R137